.github/CONTRIBUTING.md

```diff
@@ -129,18 +129,18 @@ Additional references:
   If the changes are related to an existing GitHub issue,
   please reference the issue in PR description (e.g. ```Fix #11```).
   See [this][closing-via-message] for more details.
-* If the change warrants a note in the [changelog](../CHANGELOG.MD)
-  either update the changelog in your pull request or
-  add a comment in the PR description saying that the change may warrant a note in the changelog.
-  New changes always go into the **Unreleased** section.
+
+#### Pull request - Change log
+
+* All PRs must either update the [changelog](../CHANGELOG.MD) in your pull request.
+  New changes always go into the **Unreleased** section at the top of the changelog.
```

Comment:
What about calling out `Breaking Changes`?  Is categorization of the changes handled as part of the PR update or continue to be done by maintainers?  I think ideally, the PR author should make a call and PS-Maintainers and PS-Committee should review before publishing.

Reply:
Resolved